# This message stores the bounding box coord of a detected object, based on the map frame.
# [x_min, y_min] : coordinates of the top left corner
# [x_max, y_max] : coordinates of the bottom right corner

std_msgs/Int64 x_min
std_msgs/Int64 y_min
std_msgs/Int64 x_max
std_msgs/Int64 y_max
